README.rst

```diff
@@ -10,18 +10,18 @@ queuelib
    :alt: Coverage report
 
 
-Queuelib is a collection of persistent (disk-based) queues for Python.
+Queuelib is a Python library with minimal dependencies that implements
+object collections that are persisted to disk, provide a simple API, and
+run fast.
 
-Queuelib goals are speed and simplicity. It was originally part of the `Scrapy
-framework`_ and stripped out on its own library.
+It provides collections for queues_ (FIFO), stacks_ (LIFO), queues sorted by
+priority and queues that are emptied in a round-robin_ fashion.
 
-Note: Queuelib isn't thread-safe.
+.. note:: Queuelib collections are not thread-safe.
 
-Requirements
-============
-
-* Python 2.7 or Python 3.3
```

Comment:
@Gallaecio why did you deleted the Python2 and Python3 support note? :thinking:

Reply:
Because it was not accurate (we also support Python 3.4+), and since we support all maintained Python versions I think saying “Python library” at the beginning is enough.

Reply:
It's ok. IMHO I think it would better to explicitly indicate that it supports both Python versions, but of course it's not necessary.